jdat_notebooks/niriss_ami_binary/1_niriss_ami_binary.ipynb

```diff
@@ -0,0 +1,645 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "<a id=\"top\"></a>\n",
+    "## NIRISS AMI simulation of binary point source AB-Dor and calibrator HD37093\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "skip"
+    }
+   },
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Imports\n",
+    "Describe the libraries we're using here. If there's something unusual, explain what the library is, and why we need it.\n",
+    "- *numpy* to handle array functions\n",
+    "- *astropy.io fits* for accessing FITS files\n",
+    "- *matplotlib.pyplot* for plotting data\n",
+    "- *zipfile* for accessing zip files\n",
+    "- *urllib.request* to access URL\n",
+    "- *yaml* to create yaml files\n",
+    "- *mirage* to simulate JWST data\n",
+    "- *IPython.display Image* to display png files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import io\n",
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "import numpy as np\n",
+    "from astropy.io import fits\n",
+    "import yaml\n",
+    "import zipfile\n",
+    "import urllib.request\n",
+    "from IPython.display import Image\n",
+    "\n",
+    "%matplotlib inline\n",
```

Comment:
I was able to get plt.imshow to work correctly when I moved these two lines below the mirage import lines (and then deleted the matplotlib lines in the cell containing plt.imshow)